src/klib/kvec.h

```diff
@@ -107,6 +107,7 @@
 #define kv_concat_len(v, data, len) \
   do { \
     kv_ensure_space(v, len); \
+    assert((v).items); \
```

Comment:
hmm, `v.items` can be null here if you concat `len==0` data to a zero size kvec. `memcpy()` would be UB then I think, so maybe there should be a `if (len > 0)` around the entire thing (which if coverity is good won't need yet any check)?